src/assets/translations/en/main.json

```diff
@@ -1072,7 +1072,7 @@
     "status": {
       "open": "Open",
       "fulfilled": "Fulfilled",
-      "cancelled": "Cancelled",
+      "cancelled": "Canceled",
```

Comment:
The key must not change, it is keyed by the cowswap enum with is coupled to the api response.

If we dont want it tightly coupled we should change this, but this was requested in previous code review feedback